main/src/library/Option.flix

```diff
@@ -0,0 +1,109 @@
+// TODO: List[A] vs List[a] vs List[Color] ...
+
+
+    ///
+    /// Returns `true` iff `o` is `None`.
+    ///
+    def null[a](o: Option[a]): Bool = match o with {
+        case None => true
+        case Some(v) => false
+    }
+
+    ///
+    /// Returns `v` if `o` is `Some(v)`. Undefined semantics if `o` is `None`.
+    ///
+    def get[a](o: Option[a]): a = match o with {
+        case Some(v) => v
+    }
+
+    ///
+    /// Returns `v` if `o` is `Some(v).` Otherwise returns `d`.
+    ///
+    def getWithDefault[a](o: Option[a], d: a): a = match o with {
+        case None => d
+        case Some(v) => v
+    }
+
+    ///
+    /// Returns the value of `f(v)` if `o` is `Some(v)`. Otherwise returns `false`.
+    ///
+    def exists[a](f: a -> Bool, o: Option[a]): Bool = getWithDefault(map(f, o), false)
+
+    ///
+    /// Returns the value of `f(v)` if `o` is `Some(v)`. Otherwise returns `true`.
```

Comment:
Returns `true` if `o` is `Some(v)` and the predicate `f(v)` evaluates to `true` or if `o` is `None`. Otherwise returns `false`.